src/Ganeti/HTools/CLI.hs

```diff
@@ -259,6 +261,7 @@ defaultOptions  = Options
   , optNodeSim     = []
   , optNodeTags    = Nothing
   , optSkipNonRedundant = False
+  , optStaticKvmNodeMemory = 4096
```

Comment:
Just rememberd, the default value in 2.17 is also 4G: https://github.com/ganeti/ganeti/blob/stable-2.17/src/Ganeti/ConstantUtils.hs#L236